mmf/utils/file_io.py

```diff
@@ -6,10 +6,11 @@
 
 
 try:
-    from fvcore.common.file_io import PathManager as FVCorePathManager
-except ImportError:
-    FVCorePathManager = None
+    from iopath.common.file_io import PathManager as ioPathManagerClass
```

Comment:
We will need to add `iopath` to the `requirements.txt` file as it is open sourced now. Then this try-except block can be removed. You can fix the iopath to the version you are testing with now. Also let's import to Phabricator to see if all the internal tests pass.

Reply:
I updated `requirements.txt` to reflect the new package and updated `file_io.py` to get rid of the try-except block. Then I merged the two commits into one. Now it is in Phabricator but I am not sure why tests would be failing.